packages/app-bridge/src/types/Template.ts

```diff
@@ -1,32 +1,16 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-export type TemplateApi = {
-    asset_created: string;
-    asset_modified: string;
-    categories: unknown[];
-    description: string;
-    height: number;
+export type Template = {
     id: number;
     name: string;
-    preview: string;
-    project: number;
-    project_name: string;
-    project_type: string;
-    published: number;
-    screen_id: number;
-    sector: string;
-    token: string;
-    unit: string;
-    width: number;
+    description: string;
+    projectId: number;
+    previewUrl: string;
+    pages: TemplatePage[];
 };
 
-export type Template = {
-    id: number;
-    title: string;
-    description: Nullable<string>;
-    previewUrl: string;
-    projectId: number;
-    height: number;
+export type TemplatePage = {
+    preview_url: string; // TODO: fix name inconsistency (Template.previewUrl)
```

Comment:
Please map the `underscore_case` field to `camelCase`
For typing, you can do `TemplatePageApi` which can have underscore and then `type TemplatePage = CamelCasedPropertiesDeep<TemplatePage>`
And then you would need to actually map the data where you have the repository call (`web-app`) and use `convertObjectCase(result.data, 'camel');`